examples/e2e-tests/kubernetes/coreos/coreos.json

```diff
@@ -0,0 +1,126 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "kubernetesConfig": {
+        "clusterSubnet": "10.239.0.0/16",
+        "addons": [
+          {
+            "name": "tiller",
+            "enabled": true,
+            "config": {
+              "max-history": "10"
+            },
+            "containers": [
+              {
+                "name": "tiller",
+                "cpuRequests": "1",
+                "memoryRequests": "1Gi",
+                "cpuLimits": "1",
+                "memoryLimits": "1Gi"
+              }
+            ]
+          },
+          {
+            "name": "kubernetes-dashboard",
+            "enabled": true,
+            "containers": [
+              {
+                "name": "kubernetes-dashboard",
+                "cpuRequests": "50m",
+                "memoryRequests": "512Mi",
+                "cpuLimits": "50m",
+                "memoryLimits": "512Mi"
+              }
+            ]
+          },
+          {
+            "name": "rescheduler",
+            "enabled": true,
+            "containers": [
+              {
+                "name": "rescheduler",
+                "cpuRequests": "20m",
+                "memoryRequests": "200Mi",
+                "cpuLimits": "20m",
+                "memoryLimits": "200Mi"
+              }
+            ]
+          },
+          {
+            "name": "keyvault-flexvolume",
+            "enabled": false,
+            "cpuRequests": "20m",
+            "memoryRequests": "200Mi",
+            "cpuLimits": "20m",
+            "memoryLimits": "200Mi"
+          },
+          {
+            "name": "blobfuse-flexvolume",
+            "enabled": false,
+            "cpuRequests": "20m",
+            "memoryRequests": "200Mi",
+            "cpuLimits": "20m",
+            "memoryLimits": "200Mi"
+          }
+        ]
+      }
+    },
+    "masterProfile": {
+      "count": 3,
+      "dnsPrefix": "",
+      "vmSize": "Standard_D2_v2",
+      "OSDiskSizeGB": 200,
+      "vnetSubnetId": "/subscriptions/SUB_ID/resourceGroups/RG_NAME/providers/Microsoft.Network/virtualNetworks/VNET_NAME/subnets/SUBNET_NAME",
+      "firstConsecutiveStaticIP": "10.239.255.239",
+      "vnetCidr": "10.239.0.0/16",
+      "distro": "coreos"
+    },
+    "agentPoolProfiles": [
+      {
+        "name": "agentmd",
+        "count": 3,
+        "distro": "coreos",
+        "vmSize": "Standard_D2_v2",
+        "OSDiskSizeGB": 200,
+        "storageProfile": "ManagedDisks",
+        "diskSizesGB": [
+          128,
+          128,
+          128,
+          128
+        ],
+        "availabilityProfile": "AvailabilitySet",
+        "vnetSubnetId": "/subscriptions/SUB_ID/resourceGroups/RG_NAME/providers/Microsoft.Network/virtualNetworks/VNET_NAME/subnets/SUBNET_NAME"
+      },
+      {

```

Comment:
Let's get rid of this "StorageAccount" agent pool. In the near future we're going to do some work to deprecate support for non-managed disks in AKS Engine.

Reply:
removed